README.md

```diff
@@ -332,7 +332,6 @@ In addition, runtime support is provided for:
 | Debian   | 8, 9     | x86_64 |
 | RHEL     | 6, 7     | x86_64 |
 | Ubuntu   | 12.04+   | x86_64 |
-| Windows  | 7+       | x86_64 |
```

Comment:
Should this be changed to `8+` rather than removing this Windows row?

Reply:
@zenspider ^ ?

Reply:
I would think so.